lib/ios/RNNBridgeManager.m

```diff
@@ -104,7 +109,10 @@ - (void)onJavaScriptLoaded {
 }
 
 - (void)onBridgeWillReload {
-	UIApplication.sharedApplication.delegate.window.rootViewController =  nil;
+	[_overlayManager dismissAllOverlays];
+	[_modalManager dismissAllModalsSynchronosly];
```

Comment:
Unless we also have an async method of dismissing all overlays, I think `dismissAllModals` is a more suitable name :)